src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/CaseOutcomeMap.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import static java.util.Objects.isNull;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Getter
+@AllArgsConstructor
+public enum CaseOutcomeMap {
+    REFERENCE_REVISED_FAVOUR_APPELLANT("referenceRevisedFavourAppellant", "21"),
```

Comment:
if this is only used in caseMigration, I don't think you need to put it here in common. you can just add it to case-migration code